src/concat_text/config.vsh.yaml

```diff
@@ -0,0 +1,46 @@
+name: concat_text
+description: |
+  concatenate a number of text files, handle gzipped text files gracefully and
+  optionally gzip the output text file.
+
+  This component is useful for concatening fastq files from different lanes, for instance.
+argument_groups:
+  - name: Input arguments
+    arguments:
+      - name: --input
+        description: A list of (gzipped) text files.
+        type: file
+        multiple: true
+        required: true
+        example: input?.txt.gz
+  - name: Output arguments
+    arguments:
+      - name: "--zip_output"
```

Comment:
I would propose to create an argument `--compression` in order to specify the compression algorithm to use.  This allows other algorithms to be added in the future

If we choose not to, `zip_output` is also confusing; I would use `gzip_output`

Reply:
I suggest to add some points of improvement to an info field or something:

1. What if not all of the input files are zipped or not?
2. Add compression algorithms

WDYT?

Reply:
Sure, also possible! (Could also just add a YAML comment to the config )